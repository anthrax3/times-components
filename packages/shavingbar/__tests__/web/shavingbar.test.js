import { render } from "enzyme";
import Shavingbar from "../..";

describe("Shavingbar test on web: ", () => {
  it("should render correctly", () => {});
});
