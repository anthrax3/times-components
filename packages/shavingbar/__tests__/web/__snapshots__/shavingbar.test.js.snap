// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[object Object] 1`] = `null`;

exports[`Shavingbar test on web:  should invoke callback when bubble is clicked 1`] = `
Array [
  Object {
    "hover": true,
    "isActive": false,
  },
]
`;
